doc/Developing.md

```diff
@@ -512,7 +512,7 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
  - Install `yarn`: `scoop install yarn`.
  - If you need to install `windows-build-tools`, see [`Installing Windows Build Tools`](#installing-windows-build-tools).
  - If you run into problems with installing the required build tools, the `node-gyp` documentation offers a useful [guide](https://github.com/nodejs/node-gyp#on-windows) how to install the dependencies manually. The versions required for building Theia are:
-   - Python 3.6 or higher
+   - Python 3.6 up to 3.11.x
```

Comment:
3.11.x does not work, actually: we recently had do update our ci scripts to use 3.11.1. See https://github.com/eclipse-theia/theia/pull/13011